frontend/src/components/Layout.tsx

```diff
@@ -20,20 +14,14 @@ export default function Layout({ children, locale }: Props) {
   setRequestLocale(locale);
 
   const t = useTranslations();
-  const messages = useMessages();
 
   return (
     // Stick the footer to the bottom of the page
     <div className="display-flex flex-column minh-viewport">
       <a className="usa-skipnav" href="#main-content">
         {t("Layout.skip_to_main")}
       </a>
-      <NextIntlClientProvider
```

Comment:
this change is unrelated but at one point I was messing with this file a lot and removing this unnecessary code simplified things for me a bit

Reply:
Yeah, good catch, this is already accounted for in the [parent component](https://github.com/HHS/simpler-grants-gov/blob/main/frontend/src/app/%5Blocale%5D/layout.tsx#L94)